projects/RabbitMQ.Client/client/impl/ModelBase.cs

```diff
@@ -199,14 +199,12 @@ protected void TakeOver(ModelBase other)
             _recoveryWrapper.Takeover(other._recoveryWrapper);
         }
 
-        public Task Close(ushort replyCode, string replyText, bool abort)
+        public void Close(ushort replyCode, string replyText, bool abort)
         {
-            return Close(new ShutdownEventArgs(ShutdownInitiator.Application,
-                replyCode, replyText),
-                abort);
+            _ = CloseAsync(new ShutdownEventArgs(ShutdownInitiator.Application, replyCode, replyText), abort);
```

Comment:
Is it safe to drop the task here because it means we are no longer materializing any exceptions or waiting for the operation to complete

was this only ever called by the upper layers that are now in the extension methods?

Reply:
Correct. But I agree that this should probably be fixed in another PR. (I didn't want to mix cleanup and fixing in a pr)

Reply:
Also the visibility is why I renamed it to CloseAsync and explicitely dropped the task with _ =. Glad it already worked out😛